blocks/init/src/blocks/components/page-overlay/components/link-editor.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+
+export const LinkEditor = (props) => {
```

Comment:
what is this used for in page-overlay?

Reply:
Accidentally added, removed. 👍

Reply:
Are you doing PR reviews from a van? 🤣 🚗